### ISSUE TITLE
Kernel panics on Linux

### ISSUE THREAD

**Issue by julianoes** [kevinmehall/nano-dm#2]
I have experienced system hangs (presumably kernel panics) when using nano-dm on my Linux Ubuntu 15.10 laptop. I'll need to dig into system logs though to find out more.

@kevinmehall any suggestions what to look out for? I'm assuming everything USB related?

**Comment by kevinmehall** [comment]
Interesting. I was seeing kernel panics in VirtualBox with mini-dm, and assumed it was a bug in VirtualBox USB passthrough. nano-dm is probably triggering the same bug, and if you're running Ubuntu natively, it must be a kernel bug. The kernel panics I saw in VirtualBox looked like heap corruption; the stack traces didn't always involve USB functions.

**Comment by julianoes** [comment]
I think you are right that it must be a kernel bug because I'm getting the kernel panics with mini-dm as well.

**Comment by julianoes** [comment]
Yay, the kernel bug seems to fixed with the last update for Ubuntu 15.10. No more kernel panics so far :smile:.

```
uname -a
Linux samsung 4.2.0-36-generic #42-Ubuntu SMP Thu May 12 22:05:35 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
```
